Include/objimpl.h

```diff
@@ -255,7 +255,7 @@ typedef union _gc_head {
         union _gc_head *gc_prev;
         Py_ssize_t gc_refs;
     } gc;
-    double dummy;  /* force worst-case alignment */
+    long double dummy;  /* force worst-case alignment (16 byte for amd64) */
```

Comment:
Can you please elaborate the comment? Add "bpo-27987" and explain that x64-64 ABI requires to align on 16 bytes.

Do you *have to* align to 16 bytes on x86 (32-bit) as well?

Reply:
> Can you please elaborate the comment? Add "bpo-27987" and explain that x64-64 ABI requires to align on 16 bytes.

I'm not sure this is really x86-64 ABI requirement...
C99 requires malloc returns memory aligned to be used for any built-in types.

> Do you have to align to 16 bytes on x86 (32-bit) as well?

 I don't think so.  long double may be aligned to 8 byte boundary.
But I am not sure.  I don't have 32bit development environment for now.